contrib/osx/notes_for_arm.md

```diff
@@ -0,0 +1,56 @@
+# Notes on building Electrum on Apple M1 OSX (ARM)
+
+Development version (git clone)
+
+1. Check out the code from GitHub:
+
+```
+$ git clone https://github.com/spesmilo/electrum.git
+$ cd electrum
+$ git submodule update --init
+```
+
+Run install (this should install dependencies):
+```
+python3 -m pip install --user -e .
+```
+
+2. pycryptodomex and cryptography and will need to be manually installed:
+
+```
+$ pip install pycryptodomex
+$ pip install cryptography
+```
+
+3. Manually install libsecp256k1 from source (https://github.com/bitcoin-core/secp256k1#build-steps) but with the following modifications: 
+
+```
+$ ./autogen.sh
+$ ./configure --enable-module-recovery
+$ make
+$ make check  # run the test suite
+$ sudo make install  # optional
+```
```

Comment:
Does [`contrib/make_libsecp256k1.sh`](https://github.com/spesmilo/electrum/blob/cb39bbbd943e19b392302a1ac557c374fcd2d36b/contrib/make_libsecp256k1.sh) not work?

Reply:
oops didn't see that. lets edit it to be explicit. Sure makes the next person's life a lot easier.